packages/framework/src/Component/Cron/CronModuleRepository.php

```diff
@@ -59,7 +59,12 @@ public function getAllScheduledCronModuleServiceIds()
             'SELECT cm.serviceId FROM ' . CronModule::class . ' cm WHERE cm.scheduled = TRUE'
         );
 
-        return array_map('array_pop', $query->getScalarResult());
+        $allScheduledCronModuleServiceIds = [];
+        foreach ($query->getScalarResult() as $row) {
+            $allScheduledCronModuleServiceIds[] = array_pop($row);
+        }
+
+        return $allScheduledCronModuleServiceIds;
```

Comment:
What about using 

```
return $query->getSingleColumnResult();
```

instead of the re-mapping altogether?

Reply:
ah, that is a great idea :ok_hand: :smile:

Reply:
push forced :innocent: